packages/cache/src/index.js

```diff
@@ -115,46 +115,59 @@ let createCache = (options: Options): EmotionCache => {
   ) => string | void
 
   if (isBrowser) {
-    stylis.use(options.stylisPlugins)(ruleSheet)
+    let currentSheet
+    const finalizer = middleware(
```

Comment:
seems to me that we only need to accept a list of plugins and if one wants to use custom ones they should provide our default plugins (compat and prefixer) back to us as well - this solves the ordering problem, we don't need to decide anything regarding this

OTOH removeLabel + stringify + rulesheet are essential and they should always be given as last and in that order, so I think it's safe to "hardcode" them here